gradle.properties

```diff
@@ -1,16 +1,26 @@
-org.gradle.jvmargs=-Xmx4G
-org.gradle.daemon=false
-
-
-## Environment Properties
-minecraft_version=1.20.1
-minecraft_version_range=[1.20.1,1.21)
-forge_version=47.2.18
-forge_version_range=[47,)
-loader_version_range=[47,)
-mapping_channel=parchment
-mapping_version=2023.09.03-1.20.1
+# Sets default memory used for gradle commands. Can be overridden by user or command line properties.
+org.gradle.jvmargs=-Xmx1G
+org.gradle.daemon=true
+org.gradle.parallel=true
+org.gradle.caching=true
+org.gradle.configuration-cache=true
 
+#read more on this at https://github.com/neoforged/NeoGradle/blob/NG_7.0/README.md#apply-parchment-mappings
+# you can also find the latest versions at: https://parchmentmc.org/docs/getting-started
+neogradle.subsystems.parchment.minecraftVersion=1.21.1
+neogradle.subsystems.parchment.mappingsVersion=2024.11.17
+# Environment Properties
+# You can find the latest versions here: https://projects.neoforged.net/neoforged/neoforge
+# The Minecraft version must agree with the Neo version to get a valid artifact
+minecraft_version=1.21.1
+# The Minecraft version range can use any release version of Minecraft as bounds.
+# Snapshots, pre-releases, and release candidates are not guaranteed to sort properly
+# as they do not follow standard versioning conventions.
+minecraft_version_range=[1.21.1]
+# The Neo version must agree with the Minecraft version to get a valid artifact
+neo_version=21.1.214
+# The loader version range can only use the major version of FML as bounds
+loader_version_range=[1,)
 
 ## Mod Properties
 mod_id=tanukidecor
```

Comment:
`mod_version` should be `21.1.0.1`. The first half "`21.1`" is the Minecraft version, the second half "`0.1`" is the mod version in `major.minor` format.